src/Workspaces/Core/Portable/Utilities/IReadOnlyListExtensions.cs

```diff
@@ -20,6 +20,19 @@ public static T Last<T>(this IReadOnlyList<T> list)
             return list[list.Count - 1];
         }
 
+        public static int IndexOf<T>(this IReadOnlyList<T> list, T item)
+        {
+            for (int i = 0; i < list.Count; ++i)
+            {
+                if (list[i].Equals(item))
```

Comment:
nit: use DefaultEqualityComparer as this can crash if list[i] is null

Reply:
➡️ Fixed in 7acb58206b2c442fa3a22198f3901b1ae5047f44